src/lib/server/twilio.ts

```diff
@@ -229,3 +241,53 @@ export const getMsg = async (url: URL) => {
 
 	return response;
 };
+
+export async function sendNotif() {
+	const nowLocal = new Date();
+	const users = await prisma.user.findMany({
+		select: {
+			id: true,
+			phone: true,
+			reminderDatetime: true,
+			reminderIntervalDays: true,
+			timeZone: true,
+			phonePermissions: {
+				select: {
+					allowReminders: true,
+					blocked: true
+				}
+			}
+		}
+	});
+	users.forEach(async (user) => {
+		const { id, phone, reminderDatetime, reminderIntervalDays, phonePermissions, timeZone } = user;
+		const { allowReminders, blocked } = phonePermissions;
+		if (!allowReminders || blocked) return;
+
+		const options = {
+			timeZone
+		};
+
+		const formattedDate = nowLocal.toLocaleString('en-US', options);
+		const now = new Date(formattedDate);
+		const timeDifference = Math.abs(now.getTime() - reminderDatetime.getTime()); // Get the absolute time difference in milliseconds
+		const minuteInMillis = 60 * 1000; // 1 minute in milliseconds
+		if (timeDifference < minuteInMillis) {
```

Comment:
This fails to send notifications if the cron job is offline for more than 2 minutes. It would be better to send the notifications late than never. If the reminder date is in the past, and it is currently within a 30 minutes of the time of day that the reminder is set for, we should send it. For example, if the cron fails to start and isn't running on Monday, then on Tuesday the reminder will be sent during the regular time of day.